rdr_service/dao/dv_order_dao.py

```diff
@@ -260,7 +260,7 @@ def insert_biobank_order(self, pid, resource):
         obj = BiobankOrder()
         obj.participantId = int(pid)
         obj.created = clock.CLOCK.now()
-        obj.created = datetime.datetime.now()
+        obj.finalizedTime = obj.created
```

Comment:
old code vs. new code:  Is the old code ending up with obj.created = datetime.datetime.now() a bug since it's local time?   Or is it functionality equivalent on our GCP instances to utcnow() as returned by clock.CLOCK.now()?   Wondering if previously inserted biobank order records were using an inconsistent timestamp value.

Reply:
Good catch, I wasn't aware that `CLOCK.now()` did UTC while `datetime.now()` was local. It looks like the server is set to UTC. The `created` field for request logs are saved with UTC time (latest request logs in prod currently are in the 1 o'clock time). So I found a timestamp of an existing dv order in prod and matched it with with the request log entry recorded for it and the created times are within a second of eachother.